gta3_sbl/classes.db

```diff
@@ -0,0 +1,1302 @@
+; autogenerated from ../gta3/gta3.json
+
+#CLASSESLIST
+
+Audio
+Blip
+Boat
+Camera
+Car
+CarGenerator
+CatalinaHeli
+Char
+ChaseScene
+Clipboard
+Clock
+Crane
+Credits
+Cutscene
+CutsceneHead
+CutsceneObject
+Debugger
+DrugDropOff
+DrugRun
+DynamicLibrary
+File
+Fs
+Fx
+Game
+Gang
+Garage
+Hud
+ImGui
+IniFile
+KillFrenzy
+Math
+Memory
+MemoryLibrary
+Mission
+Object
+Pacman
+Pad
+Pager
+Path
+Phone
+Pickup
+Player
+Rc
+Restart
+ScriptFire
+Sound
+Sphere
+Stat
+Streaming
+StuckCarCheck
+Text
+Txd
+Weapon
+Weather
+World
+Zone
+
+#CLASSES
+$
```

Comment:
Is orphan `$` is supposed to be here?

Reply:
No, it's not. Library generator adds empty class for non-class commands. I may look at it at some point, but I don't think it affects anyone

Reply:
fixed